src/main/java/core/basesyntax/strategy/impl/NewYearDiscountService.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.strategy.DiscountService;
+
+public class NewYearDiscountService implements DiscountService {
+    private static final double DISCOUNT_PERCENT = 20;
```

Comment:
```suggestion
    private static final double NEW_YEAR_DISCOUNT = 20;
```
аналогічно в інших класах для інформативності